packages/client/src/pages/forum/topicWithComments/page/page.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import style from './page.module.scss';
+import Pagination from '@/components/ui/pagination/pagination';
+import { TopicInfo } from '../topicInfo/topicInfo';
```

Comment:
Можно добавить файл index.ts в каждой директории и экспортировать оттуда чтобы импорты были более красивыми:
```suggestion
import Pagination from '@/components/ui/pagination';
import { TopicInfo } from '../topicInfo';
```